tests/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-access-control -DJUCI_BUILD_PATH=\\\"${CMAKE_BINARY_DIR}\\\" -DJUCI_TESTS_PATH=\\\"${CMAKE_CURRENT_SOURCE_DIR}\\\"")
 
 set(stub_files
-    stubs/config.cc
-    stubs/dialogs.cc
-    stubs/directories.cc
-    stubs/info.cc
-    stubs/selectiondialog.cc
-    stubs/terminal.cc
-    stubs/tooltips.cc
+    ${CMAKE_CURRENT_SOURCE_DIR}/stubs/config.cc
+    ${CMAKE_CURRENT_SOURCE_DIR}/stubs/dialogs.cc
+    ${CMAKE_CURRENT_SOURCE_DIR}/stubs/directories.cc
+    ${CMAKE_CURRENT_SOURCE_DIR}/stubs/info.cc
+    ${CMAKE_CURRENT_SOURCE_DIR}/stubs/selectiondialog.cc
+    ${CMAKE_CURRENT_SOURCE_DIR}/stubs/terminal.cc
+    ${CMAKE_CURRENT_SOURCE_DIR}/stubs/tooltips.cc
```

Comment:
Great, but just a minor comment: `${CMAKE_CURRENT_SOURCE_DIR}` is used here, but not in: https://github.com/cppit/jucipp/pull/282/commits/425f8ef52baaf89e142c6b9cf7796da1266ea723#diff-95e351a3805a1dafa85bf20b81d086e6L33. I guess `stubs/config.cc` here for instance is ok instead of `${CMAKE_CURRENT_SOURCE_DIR}/stubs/config.cc`, or what do you think?